packages/xrpl/typedoc.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
```

Comment:
Can we leave a TODO here (or open an issue) to switch back to TS?

Reply:
@mvadari  Is there a way to switch back to typescript without switching from `commonjs` -> ESM ?

https://github.com/TypeStrong/typedoc/discussions/2870#discussioncomment-12333395